spec/support/idv_examples/max_attempts.rb

```diff
@@ -1,78 +1,33 @@
-shared_examples 'idv max step attempts' do |sp|
-  it 'allows 3 attempts in 24 hours', :email do
-    visit_idp_from_sp_with_loa3(sp)
-    user = register_user
+shared_examples 'verification step max attempts' do |step|
+  scenario 'more than 3 attempts in 24 hours prevents further attempts' do
+    visit_idp_from_sp_with_loa3(:oidc)
 
-    max_attempts_less_one.times do
-      visit verify_session_path
-      fill_out_idv_form_fail
-      click_idv_continue
-
-      expect(current_path).to eq verify_session_result_path
```

Comment:
Overall, this looks great. I like the cleanup. I noticed a few expectations were removed, like this one and other similar ones, and also another one that checks errors on the page, such as `expect(page).to have_content(t('idv.errors.hardfail'))`. Was that intentional?

Reply:
Yeah, let's see. Looks like I left off checking the path, the idv columns on the user, and attempting to re-enter the step. I'll re-add those.

Reply:
Oh yeah, also on this line I removed it because it doesn't make sense to me why it is here. You wouldn't be able to complete the form if you were looking at the right form.

Reply:
Also looking now, I do have a spec that specifically covers trying to re-enter the step after failing and it does check for the hardfail error message.